### ISSUE TITLE
Create Pulsar Topology Frontend

### ISSUE THREAD

**Issue by MPSebastian** [amosproj/amos2023ss01-apache-pulsar-ui#28]
## User story
1. As a user
2. I want to have a well structured overview of all my pulsar components
3. So that I can interactively browse through my hirarchy

## Acceptance criteria
* [ ] Users can view the topology of an Apache Pulsar installation (multi-tenant, namespaces, multi-clusters)
* [ ] User can view all topics
* [ ] User can view all messages
* [ ] User can interactively browse its different levels

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
